tests/test_package/test_log.py

```diff
@@ -89,49 +93,166 @@ def test_logging_warning_capture():
     monitor_flux = td.FluxMonitor(
         center=(0, 0, 0),
         size=(8, 8, 8),
-        freqs=freqs,
+        freqs=list(freqs),
```

Comment:
did this not work with `freqs` passed as a numpy array? I would have thought ArrayLike would handle this?

Reply:
no, I was just playing around with adding frequencies outside of source range (something like `freqs=list(freqs) + [1]`), and this remained like that accidentally